integration/apps/sinatra2_classic/docker-compose.yml

```diff
@@ -0,0 +1,80 @@
+version: '3.4'
+services:
+  app:
+    build:
+      context: .
+      args:
+        BASE_IMAGE: datadog/dd-apm-demo:rb-2.7
+    depends_on:
+      - ddagent
+    environment:
+      - BUNDLE_GEMFILE=/app/Gemfile
+      - DD_AGENT_HOST=ddagent
+      - DD_METRIC_AGENT_PORT=8125
+      - DD_TRACE_AGENT_PORT=8126
+      - DD_HEALTH_METRICS_ENABLED=true
+      - DD_SERVICE=acme-sinatra
+      - DD_PROFILING_ENABLED=true
+      # Use these to choose what is run
+      - DD_DEMO_ENV_PROCESS=puma
+      - DD_DEMO_ENV_FEATURES=tracing,profiling
+      # Use this for a local version of ddtrace
+      - DD_DEMO_ENV_GEM_LOCAL_DDTRACE=/vendor/dd-trace-rb
+      # Use these for a specific revision of ddtrace
+      # - DD_DEMO_ENV_GEM_GIT_DDTRACE=https://github.com/DataDog/dd-trace-rb.git
+      # - DD_DEMO_ENV_GEM_REF_DDTRACE=f233336994315bfa04dac581387a8152bab8b85a
+      # Enable building the profiling native extension
+      # - DD_DEMO_ENV_BUILD_PROFILING_EXTENSION=true
+    expose:
+      - "80"
+    stdin_open: true
+    tty: true
+    volumes:
+      - .:/app
+      - ./data/app:/data/app
+      - bundle:/usr/local/bundle
+      - ../../images/include:/vendor/dd-demo
+      - ../../..:/vendor/dd-trace-rb
+  ddagent:
+    image: datadog/dd-apm-demo:agent
+    environment:
+      - DD_APM_ENABLED=true
+      - DD_PROCESS_AGENT_ENABLED=false
+      - DD_BIND_HOST=0.0.0.0
+      - DD_API_KEY
+      - LOG_LEVEL=DEBUG
+      - DD_LOGS_STDOUT=yes
+      - DD_DOGSTATSD_NON_LOCAL_TRAFFIC=true
+    expose:
+      - "8125/udp"
+      - "8126"
+    volumes:
+      - ../../images/agent/agent.yaml:/etc/datadog-agent/datadog.yaml
+      # For monitoring performance of containers (e.g. CPU, Memory, etc...)
+      # - type: bind
+      #   source: ../../images/agent/agent.yaml
+      #   target: /etc/datadog-agent/datadog.yaml
+      # - type: bind
+      #   source: /var/run/docker.sock
+      #   target: /var/run/docker.sock:ro
+      # - type: bind
+      #   source: /proc/
+      #   target: /host/proc/:ro
+      # - type: bind
+      #   source: /sys/fs/cgroup/
+      #   target: /host/sys/fs/cgroup:ro
+  load-tester:
+    build:
+      context: ../../images
+      dockerfile: wrk/Dockerfile
+    command: -t10 -c10 -d43200s -s /scripts/scenarios/basic/default.lua http://app/basic/fibonacci
```

Comment:
Pointing the load tester at the fibonacci example may give a bit inconsistent results, since the workload is variable. Was this intended?